libs/services/Converter.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+
+class Converter:
+    def __init__(self):
+        # used to store individual API calls to avoid executing
+        # the same query multiple times in single session
+        self.cache = dict()
+
+    def connect_to_service(self, service, args):
+        """
+        Make get request to given service with arguments.
+        Raises ConnectionError if service is not available.
+
+        :param service: requested service to be queried
+        :param args: additional query arguments
+        :return: obtained response
+        """
+        try:
+            identification = service + ":" + args
```

Comment:
Let's use f-strings everywhere, because they are honking awesome! 

https://docs.python.org/3.6/reference/lexical_analysis.html#formatted-string-literals
https://www.python.org/dev/peps/pep-0498

Reply:
That is good point, I'm still not used to `f-strings` from previous versions of Python.
I changed all strings manipulation to `f-strings` in 66c4434.